e2e/atlas/datalake_test.go

```diff
@@ -94,17 +91,7 @@ func TestDatalake(t *testing.T) {
 		resp, err := cmd.CombinedOutput()
 
 		if err != nil {
-			t.Errorf("unexpected error: %v, resp: %v", err, string(resp))
-		}
-	})
-
-	t.Run("Delete", func(t *testing.T) {
-		cmd := exec.Command(cliPath, atlasEntity, datalakeName, "delete", datalakeName)
```

Comment:
this was the error it should have been 
`cliPath, atlasEntity, datalakeEntity, "delete", datalakeName, "--force"`